widgets/features/styles/default.less

```diff
@@ -17,25 +17,162 @@
 @more_text_size: default;
 @more_text_color: default;
 
-.sow-features-list .sow-features-feature {
-
-   .textwidget {
-       > h5 {
-           .font(@title_font, @title_font_weight);
-           font-size: @title_size;
-           color: @title_color;
-       }
-
-       > p {
-           .font(@text_font, @text_font_weight);
-           font-size: @text_size;
-           color: @text_color;
-       }
-
-       > p.sow-more-text {
-           .font(@more_text_font, @more_text_font_weight);
-           font-size: @more_text_size;
-           color: @more_text_color;
-       }
-   }
+@container_size: default;
```

Comment:
Working from memory here, so I could be wrong, but the main intention with default was that if an attribute has the value `default`, then it's removed from the CSS. You can see that in action here - https://github.com/siteorigin/so-widgets-bundle/blob/develop/base/siteorigin-widget.class.php#L853

We're probably pretty safe with `@container_size`, because it's a measurement field, but technically if this is ever empty and the value of `@container_size` stays as `default`, then we might have problems with things like `padding-right: @container_size + 10px;`, which would be the same as `padding-right: default + 10px;`.

Because default doesn't really serve much of a purpose here, we could enter a safer default, or just use the main widget's default.